tests/golden/defaults/appuio-cloud/appuio-cloud/01_agent/10_webhook_config.yaml

```diff
@@ -0,0 +1,33 @@
+      service:
+        name: webhook-service
+        namespace: appuio-cloud
+        path: /validate-request-ratio
+    failurePolicy: Ignore
+    matchPolicy: Equivalent
+    name: validate-request-ratio.appuio.io
+    namespaceSelector:
+      matchExpressions:
+        - key: appuio.io/organization
+          operator: Exists
+    rules:
+      - apiGroups:
+          - '*'
+        apiVersions:
+          - '*'
+        operations:
+          - CREATE
+          - UPDATE
+        resources:
+          - '*'
```

Comment:
Why not only pods/... ?

Reply:
The idea here was that if a user exceeds the fair use limit they will get a warning for pretty much every action on the cluster, so that they do not miss it.  See [APUB 262](https://ticket.vshn.net/browse/APUB-262) (internal)

Reply:
Oh ok 👍